src/TLuaInterpreterMapper.cpp

```diff
@@ -3398,6 +3399,7 @@ int TLuaInterpreter::setExitWeight(lua_State* L)
 
     pR->setExitWeight(direction, weight);
     lua_pushboolean(L, true);
+    host.mpMap->update();
```

Comment:
Setting an exit weight does not produce any visible changes - so this is not needed.

Reply:
See https://github.com/Mudlet/Mudlet/pull/7187#discussion_r1544529665